README.md

```diff
@@ -73,6 +73,10 @@ Download a release of the toolchain for your platform from [Github
 releases](https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/releases)
 and extract the archive into an arbitrary directory.
 
+### Pre-requisite to use toolchain in Windows 10 and Windows 11
```

Comment:
It looks like "on Windows" is more commonly used?

Reply:
Fixed in latest version of patch.

Reply:
Fixed in latest version of patch.